Source/WebCore/rendering/RenderDeprecatedFlexibleBox.cpp

```diff
@@ -137,6 +137,13 @@ RenderDeprecatedFlexibleBox::RenderDeprecatedFlexibleBox(Element& element, Rende
     m_stretchingChildren = false;
 }
 
+RenderDeprecatedFlexibleBox::RenderDeprecatedFlexibleBox(Document& document, RenderStyle&& style)
+    : RenderBlock(RenderObject::Type::DeprecatedFlexibleBox, document, WTFMove(style), { })
+{
+    setChildrenInline(false); // All of our children must be block-level
+    m_stretchingChildren = false;
+}
```

Comment:
Is there a way to avoid the duplication with the other constructor? Same question for the other classes.